008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1021 @@
+
+\subsection{Initialisation}
+The agent takes a simple algorithmic approach to formulating a social network; firstly, the ID of the neighbouring agents is determined through message passing, this ID is mapped to an initial trust value and the trust value is continually updated through subsequent social interactions. 
+
+This agent implementation offers different initialised trust values based on the social motive, using the general notion that the more narcissistic an agent is, the less likely it is to have an initial positive opinion of their neighbours. 
+
+For this reason, we give a maximal range of trust from -25 to +25 with the initialisations as follows:
+
+\begin{itemize}
+    \item Narcissist: -5
+    \item Selfish: 0
+    \item Collectivist: +5
+    \item Altruist: +10
+\end{itemize}
+
+\subsection{Causes for Trust Update}
+
+It is through continuous social interaction in the tower that agents will be able to form opinions of the other agents. For this agent, there is a specific set of interactions that will result in a opportunity for trust to update. These interactions impose a \textit{base} behavioural update, before scaling due to social motive (\Cref{sec:varTrust}).
+
+\subsubsection{Treaties}
+
+\begin{enumerate}
+    \item When an agent receives a response to the treaty it has proposed: 
+    \begin{itemize}
+        \item If accepted: +3
+        \item If rejected: -2
+    \end{itemize}
+    \item When an agent receives a treaty proposal: 
+    \begin{itemize}
+        \item  If the treaty is evaluated as good: +2
+        \item   If the treaty is evaluated as bad: -1
+    \end{itemize}
+\end{enumerate}
+
+\subsubsection{Communications}
+
+\begin{enumerate}
+    \item If an agent makes a request for another to take 0 food (\Cref{sec:takeNone}):
+    \begin{itemize}
+        \item  If accepted: reset to 0
+        \item  If rejected: -1
+    \end{itemize}
+    \item If an agent is requested to leave food: 
+    \begin{itemize}
+        \item If narcissist: -1
+        \item Otherwise: +0
+    \end{itemize}
+    \item If an agent is requested to take a certain amount of food: -1
+\end{enumerate}
+
+\subsubsection{Case of Low Trust} \label{sec:takeNone}
+
+If an agent holds an extremely low opinion of another agent (trust $\leq$ -10), it will send a message asking the other to take 0 food for 1 turn, so as to restore trust and show repentance. If this agent accepts the request, it is forgiven and have its trust value reset to 0. If not, the opinion of this agent continues to decrease (-1).
+
+\subsubsection{Case of Low Average Food} \label{sec:lowFood}
+
+If an agent continually averages less that 1 unit of food per day, it makes the assumption that the agent above is acting maliciously. For this reason, the trust of this neighbouring agent is decreased (-1). Conversely, if the agent is averaging above this amount, it will increase the trust in the neighbour above (+1).
```

Comment:
```suggestion
If an agent continually averages less than one unit of food per day, it makes the assumption that the agent above is acting maliciously. For this reason, the trust of this neighbouring agent is decremented. Conversely, if the agent is averaging above this amount, it will increment its trust in the neighbour above.
```